ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py

```diff
@@ -1560,7 +1560,7 @@ def test_tensorflow_linspace(
 @handle_frontend_test(
     fn_tree="tensorflow.realdiv",
     dtype_and_x=helpers.dtype_and_values(
-        available_dtypes=helpers.get_dtypes("float"),
+        available_dtypes=helpers.get_dtypes("float", "complex"),
```

Comment:
pass it as `"float_and_complex"`, you can read the docs of `get_dtypes` for more info